pkg/vault/test/foundry/BufferVaultPrimitive.t.sol

```diff
@@ -146,9 +140,46 @@ contract BufferVaultPrimitiveTest is BaseVaultTest {
     /********************************************************************************
                                         Deposit
     ********************************************************************************/
-    function testDeposit() public {
+    function testDepositInteractionWithERC4626Protocol() public {
+        // Initializes the buffer with an amount that's not enough to fulfill the deposit operation, so the vault has
+        // to interact with the ERC4626 protocol.
```

Comment:
Maybe worth checking vault balances and reserves as with the DDoS test?

Reply:
Done!